components/kyma-environment-broker/hack/scripts/README.md

```diff
@@ -0,0 +1,68 @@
+## Create Network and CloudNAT
```

Comment:
Not sure this file should be here. For example, you use in the resources the domain name `*.cis.gophers.kyma.pro`. Maybe you can move this domain as an environment variable?